app/models/questionable/question.rb

```diff
@@ -20,24 +20,25 @@ def #{type}?
     end
 
     def answers_for_user(user)
-      answers = self.answers.where(user_id: user.id)
```

Comment:
Found more redundant usages of self:

https://github.com/bespokepost/questionable/blob/c0099759813d746d4569f2364ff2b0739b85d960/app/models/questionable/assignment.rb#L22

https://github.com/bespokepost/questionable/blob/c0099759813d746d4569f2364ff2b0739b85d960/app/models/questionable/assignment.rb#L27